packages/contracts-bedrock/test/L2/L1Block.t.sol

```diff
@@ -57,7 +65,9 @@ contract L1BlockBedrock_Test is L1BlockTest {
             _sequenceNumber: type(uint64).max,
             _batcherHash: bytes32(type(uint256).max),
             _l1FeeOverhead: type(uint256).max,
-            _l1FeeScalar: type(uint256).max
+            _l1FeeScalar: type(uint256).max,
+            _interopSetSize: uint8(_chainIds.length),
+            _chainIds: _chainIds
```

Comment:
Thoughts on calling this "dependency set" to match the spec?

Reply:
I'm using `_chainIds` for consistency with https://github.com/ethereum-optimism/specs/blob/main/specs/interop/predeploys.md#l1attributes (check the name of the last input arg in L1Attributes's table). However, I think "dependency set" would be more descriptive. In that case, we should also update the table so that the arg is something like `dependencySet`, right? or even `interopSet`, so that it relates to `interopSetSize` more directly. alternatively, we could update `interopSetSize` to `dependencySetSize`. thoughts? cc @protolambda

Reply:
dropped `_interopSetSize` and changed `_chainIds` for `_dependencySet`